Frontend/library/src/WebXR/WebXRController.ts

```diff
@@ -485,12 +485,12 @@ export class WebXRController {
             Logger.Info('WebXR requires https, if you want WebXR use https.');
         }
 
-        if (navigator.xr) {
-            return navigator.xr.isSessionSupported(mode);
-        } else {
-            return new Promise<boolean>(() => {
-                return false;
-            });
-        }
+        try {
```

Comment:
Adding a comment here to explain try-catch usage

```suggestion
        // Wrap in try-catch because access to XR object can be denied due
        // to browser security permissions (e.g. streaming from an iframe)
        try {
```